modules/core/src/main/java/org/apache/ignite/internal/visor/snapshot/VisorSnapshotRestoreTask.java

```diff
@@ -25,8 +25,11 @@
 
 /**
  * Visor snapshot restore task.
+ *
+ * @deprecated Use {@link VisorSnapshotStatusTask} instead.
  */
 @GridInternal
+@Deprecated
```

Comment:
I think we should deprecate only job now.

Reply:
Fixed, thanks.